gulp/config.js

```diff
@@ -1,11 +1,66 @@
-'use strict';
+var dest = "./dist/assets/";
+var src = "./src/assets";
+var version = "/1.0.0";
 
 module.exports = {
+  concatJS: {
```

Comment:
There's a mixture of 4 space and 2 space indentation in these files. We favor 4 spaces for JS/CSS, and 2 spaces for HTML. Using editorconfig (as mentioned earlier) will help your editor follow this rule automatically.